### PR TITLE
[pt] Add content for hello-minikube in português

### PR DESCRIPTION
**Pull request by jailton**
Translate content/pt/docs/tutorials/hello-minikube.md in Portuguese #13939

/language pt

### CODE REVIEW

**Comment by irvifa** [comment]
/retitle [pt] Add content for hello-minikube in português

File: content/pt/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+## Objetivos
+
+* Instalar uma aplicação exemplo no minikube.
+* Executar a aplicação.
+* Visualizar os logs da aplicação.
+
+## Antes de você iniciar
+
+Este tutorial disponibiliza uma imagem de contêiner que utiliza o NGIX para retornar todas as requisições.
```
Comment (jcjesus) [inline content/pt/docs/tutorials/hello-minikube.md:32]: Hi Jailton, please, can you solve the typo in the word "NGIX" to "NGINX"?
Thanks for your contribution!

File: content/pt/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+    A saída será semelhante a:
+
+    ```
+    NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    hello-node   LoadBalancer   10.108.144.78   <pending>     8080:30369/TCP   21s
+    kubernetes   ClusterIP      10.96.0.1       <none>        443/TCP          23m
+    ```
+
+    Em provedores de Cloud que fornecem serviços de balaceamento de carga para o Kubernetes, um IP externo seria provisionado para acessar o serviço. No Minikube, o tipo `LoadBalancer` torna o serviço acessível por meio do comando `minikube service`.
```
Comment (jcjesus) [inline content/pt/docs/tutorials/hello-minikube.md:135]: Please, can you fix the word "balaceamento" to "balanceamento"?
Thanks Jailton!

File: content/pt/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+4. (**Apenas no ambiente do Katacoda**) Clicar no sinal de mais e então clicar em **Select port to view on Host 1**.
+
+5. (**Apenas no ambiente do Katacoda**) Observe o número da porta com 5 dígitos exibido ao lado de `8080` na saída do serviço. Este número de porta é gerado aleatoriamente e pode ser diferente para você. Digite seu número na caixa de texto do número da porta e clique em **Display Port**. Usando o exemplo anterior, você digitaria `30369`.
+
+Isso abre uma janela do navegador, acessa o seu aplicativo e mostra o retorno da requisição.
+
+## Habilitando Complementos (addons)
+
+O Minikube inclue um conjunto integrado de {{< glossary_tooltip text="complementos" term_id="addons" >}} que podem ser habilitados, desabilitados e executados no ambiente Kubernetes local.
```
Comment (jcjesus) [inline content/pt/docs/tutorials/hello-minikube.md:151]: Hi, here can you fix the word "inclue" to "inclui".
Thanks

**Review by jcjesus — COMMENTED** [review]
Hi Jailton, I'm Julio Cesar, it's my pleasure to know you and I'm want to say, very thanks for your contribution! In the beginning, the Portuguese version was me and Felipe and now we have more contributors, including you! Very so much thanks!
Please, can you fix the little typos to approve this MR.

**Comment by jailton** [comment]
Hi @jcjesus! It's a pleasure to contribute to the translation into Portuguese. I decided to contribute to the project after attending a talk by @bradtopol last year. :)

File: content/pt/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
+---
+title: Olá, Minikube!
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "Iniciar"
+    weight: 10
+    post: >
+      <p>Pronto para meter a mão na massa? Vamos criar um cluster Kubernetes simples e executar uma aplicacão exemplo.</p>
```
Comment (devlware) [inline content/pt/docs/tutorials/hello-minikube.md:10]: There is a typo in aplicação, there is a missing ç.

**Review by devlware — APPROVED** [review]
Thank you @jailton for your contribuition. I found just one small typo.

File: content/pt/docs/tutorials/hello-minikube.md (hunk 4 above)
Comment (jailton) [reply content/pt/docs/tutorials/hello-minikube.md]: HI @devlware! Thanks for the review.

**Pull request merged**